app/scripts/controllers/createPersistentVolumeClaim.js

```diff
@@ -49,7 +52,7 @@ angular.module('openshiftConsole')
       .then(_.spread(function(project, context) {
         $scope.project = project;
 
-        if (!AuthorizationService.canI('persistentvolumeclaims', 'create', $routeParams.project)) {
+        if (!AuthorizationService.canI(persistentVolumeClaimsVersion, 'create', $routeParams.project)) {
```

Comment:
We should be checking the same resourceGroupVersion that we create

Reply:
Yeah that makes sense. Let me update.